src/main/java/com/epam/repository/PersonDAO.java

```diff
@@ -26,7 +26,7 @@ public PersonDAO(NamedParameterJdbcTemplate jdbcTemplate) {
     Person buildPerson(ResultSet rs) throws SQLException {
         return Person.builder()
                 .id(rs.getInt("id"))
-                .name(rs.getString("nickname"))
+                .name(rs.getString("email"))
```

Comment:
How it will used in registration? It is security functionality... If you want to fix it - please make another pull request with this hot-fix. Because, imagine, that you will fix a lot of other logic, and how the reviewers will find your task logic?